recipes/quazip/all/test_package/CMakeLists.txt

```
@@ -1,13 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```
Please update `cmake_minimum_required`.
`target_compile_features` requires cmake/3.8.